DEVELOP.md

```diff
@@ -0,0 +1,294 @@
+Rig Development
+===============
+
+This document presents an overview of how to get up and running with a
+development install of Rig and how it is tested, reviewed and released.
+
+* [Developer Installation](#developer-installation)
+* [Testing](#testing)
+* [Documentation](#documentation)
+* [Versioning and Releasing](#versioning-and-releasing)
+* [Third-Party Services](#third-party-services)
+
+
+Developer Installation
+----------------------
+
+By following the steps below, you can set up an appropriate environment for Rig
+development.
+
+### `virtualenv` setup (optional)
+
+We recommend working in a [virtualenv](https://pypi.python.org/pypi/virtualenv)
+which can be set up like so:
+
+    # `--system-site-packages` optionally allows the virtualenv to use your
+    # system-wide installations of large packages (e.g. NUMPY)
+    virtualenv --system-site-packages rig_virtualenv
+    cd rig_virtualenv
+    . bin/activate
+
+`virtualenv`s provide a (sort-of) standardised, sandboxed Python environment
+for testing and working with rig. Run `. bin/activate` whenever you want to
+re-enter the `virtualenv` environment in a new terminal. To leave the
+`virtualenv` just run `deactivate`. 
+
+### Installing Rig for Development
+
+A development installation of rig can be created straight out of the repository
+using [setuptools](https://pypi.python.org/pypi/setuptools) as usual:
+
+    git clone git@github.com:project-rig/rig.git
+    cd rig
+    python setup.py develop
+
+### Reporting Issues and Making Suggestions
+
+Bug reports and suggestions can be made via the [GitHub issues
+page](https://github.com/project-rig/rig/issues).
+
+### Contributing
+
+Contributions are very much welcome to the project! To contribute, please make
+your contribution off the master branch (making sure to update the
+[`CONTRIBUTORS.md file`](CONTRIBUTORS.md)) and submit a GitHub pull request for
+review.
+
+Testing
+-------
+
+Rig includes a test suite which *aims* to comprehensively test all of its
```

Comment:
The "*aims*" worries me.  Our intent is that the tests are comprehensive and that where they are not it is equivalent to a bug.  To me that implies that we can say that our tests *are* comprehensive.

(Note we don't say that we *aim* to produce flawless code, this is a given).